src/ops/direnv/envrc.bash

```diff
@@ -169,5 +169,9 @@ elif [ -f "$EVALUATION_ROOT" ]; then
     # shellcheck disable=SC1090
     . "$EVALUATION_ROOT"
 fi
+if [ -f "$EVALUATION_ROOT/alias-export" ]; then
+    # shellcheck disable=SC1090
+    . "$EVALUATION_ROOT/alias-export"
```

Comment:
This works: aliases set in the `shellHook` get exported and sourced. Unfortunately, functions defined in the `shellHook` do not get exported. So e.g. the `ci` alias in this project gives an error under `lorri shell` as of now:

```console
$ lorri shell
$ ci
ci_check: command not found
```

Reply:
Looks great! functions are never exported with lorri, so that is expected behavior. If we want a `ci` or `ci_check` etc. function to come through, best to make it a script.

Reply:
Wait no, let's ditch this. I don't think we should support aliases: `lorri shell` will ideally use whatever the user's $SHELL is and supporting things like aliases makes that much harder.

Reply:
I've removed the alias export / import stuff.

> ideally use whatever the user's $SHELL is

Okay, right now it's using `bashInteractive` from lorri's runtime closure. Should it be using $SHELL instead? In that case we would need a variant of `envrc.bash` for every supported shell.

Reply:
It looks like `nix-shell` also drops you into `bashInteractive` regardless of the original shell.

```console
$ sh
$ ps -p $$
  PID TTY          TIME CMD
 1612 pts/0    00:00:00 sh
$ nix-shell
$ ps -p $$
  PID TTY          TIME CMD
 1638 pts/0    00:00:00 bash
```

I get the same when the starting point is a `fish` shell.